llvm/lib/Transforms/Vectorize/LoopVectorizationLegality.cpp

```diff
@@ -943,11 +955,22 @@ bool LoopVectorizationLegality::canVectorizeInstrs() {
       if (CI && !VFDatabase::getMappings(*CI).empty())
         VecCallVariantsFound = true;
 
+      auto CanWidenInstruction = [this](Instruction const &Inst) {
+        Type *InstTy = Inst.getType();
+        if (isa<CallInst>(Inst) && isa<StructType>(InstTy) &&
+            canWidenCallReturnType(InstTy)) {
+          StructVecCallFound = true;
+          // For now, we can only widen struct values returned from calls where
+          // all users are extractvalue instructions.
+          return llvm::all_of(Inst.users(), IsaPred<ExtractValueInst>);
```

Comment:
```suggestion
          return all_of(Inst.users(), IsaPred<ExtractValueInst>);
```

`llvm::` should also not be needed